MSAL/src/native_auth/public/state_machine/error/AttributesRequiredError.swift

```diff
@@ -26,11 +26,16 @@ import Foundation
 
 @objc
 public class AttributesRequiredError: MSALNativeAuthError {
+    /// Describes an error that provides messages describing why an error occurred and provides more information about the error.
```

Comment:
This string doesn't read write. Maybe it should be just `/// Describes why an error occurred and provides more information about the error.`?

Reply:
Same comment applies to many places because of the duplication.

Reply:
Yeah, it was weird. Thanks for the feedback. Updated 👍.